### DISCUSSION TITLE
[A-L] (12 Gennaio 2022) Esercizio 3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#297] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152745894-e30975a3-ca3e-43f7-9169-8f9796c443bf.png)

**Comment by i0nut02** [comment]
1. V
2. F
3. F

**Comment by Elia-Belli** [comment]
- **(Vero) Riflessività**: $(x,y)R(x,y)$, affinchè sia vero o $(x,y) \subseteq (x,y)$ o $y\< x$, siccome $(x,y)$ è sempre un sottoinsieme improprio di se stesso allora la relazione è riflessiva.

- **(Falso) Simmetria**: $(x,y)R(w,z) \Rightarrow (w,z)R(x,y)$, studiamo i vari casi:
**1)** Se $(x,y)R(w,z)$ perchè $(x,y) \subseteq (w,z)$ vale la simmetria solo se $(x,y) = (w,z)$, se $(x,y)$ fosse un insieme proprio di $(w,z)$ è impossibile che $(z \< x)$ e nemmeno che $(w,z) \subseteq (x,y)$. 
Quindi possiamo fermarci al caso in cui $(x,y)$ è sottoinsieme proprio di $(w,z)$ e osservare che non vale la simmetria.

- **(Falso) Transitività**: $(x,y)R(w,z) \land (w,z)R(h,g) \Rightarrow (x,y)R(h,g)$:
**1)** Se $y \< w \< h \Rightarrow (x,y)R(h,g)$;
**2)** Se $y \< w \land (w,z) \subseteq (h,g)$ non è assicurato che $y \< h$ e nemmeno che $(x,y) \subseteq (h,g)$, mostro un esempio: sia $(x,y) = (0,1), (w,z) = (2,3)$ e $(h,g) = (1,4)$, in questo caso $(2,3) \subseteq (1,4)$ e $y \< w$ ma non è vero che $1 \< 0$ e nemmeno $(0,1) \subseteq (1,4)$. 
Quindi possiamo fermarci a questo caso e dire che la relazione non è transitiva.
